Goobi/setup/default.sql

```diff
@@ -16,27 +16,17 @@
 /*!40000 ALTER TABLE `benutzer` DISABLE KEYS */;
 INSERT INTO `ldapgruppen` (
   `ldapgruppenID`, `titel`) VALUES 
-  (2, 'test'); /* id 2 has to be inserted to match users (benutzer)*/
+  (2, 'Example LDAP group'); /* id 2 has to be inserted to match users (benutzer)*/
 /*!40000 ALTER TABLE `benutzer` ENABLE KEYS */;
 
 --
--- Password for test users is "test"
--- NOTICE: Disable those users in production environment!
+-- Password for admin user is "goobi"
```

Comment:
Why you are using `goobi` as login password? Application and documentation is already adjusted to use Kitodo in many places.

Reply:
Changed